bin/install-studio-cli.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# This script is used to install the Studio CLI on macOS. It creates a symlink at CLI_SYMLINK_SOURCE
+# (e.g. /usr/local/bin/studio) pointing to the packaged Studio CLI JS file at CLI_SYMLINK_TARGET.
+
+# Exit if any command fails
+set -e
+
+if [ -z "$CLI_SYMLINK_SOURCE" ] || [ -z "$CLI_SYMLINK_TARGET" ]; then
+	echo "Error: CLI_SYMLINK_SOURCE and CLI_SYMLINK_TARGET environment variables must be set"
+	exit 1
+fi
+
+DIRECTORY_PATH=$(dirname "$CLI_SYMLINK_SOURCE")
+
+rm -f "$CLI_SYMLINK_SOURCE"
```

Comment:
I understand the point of `-f` is to silently ignore cases where the symlink doesn't exist yet, but I wonder if we could guard the `rm` such that it will only delete if the file is a symlink:

```sh
[ -h "$file" ] && rm "$file"
```

(yes, this is me feeling nervous about a `rm -f` command being run as root 😄)

Reply:
The path provided by Studio is static, so I struggle to see how the current logic would go wrong. However, your approach is elegant, so I don't see why not 👍

Reply:
> The path provided by Studio is static

I suppose the strictest version would be:

```sh
[ "$1" = "/usr/local/bin/studio" ] || exit 1
```

You may ask: why not just skip passing that argument, then? Because this makes it immediately obvious to any reader that this script is tied to a particular use, prompting the reader to go look for its JS counterpart.